src/LibraBFT/Prelude.agda

```diff
@@ -150,11 +150,25 @@ module LibraBFT.Prelude where
     hiding (align; alignWith; zipWith)
     public
 
-  -- a non-dependent eliminator
+  -- a non-dependent eliminator; note the traditional argument order
```

Comment:
why not a newline before `note` so the remark is all on the second line?

Reply:
DONE